canonical-web-and-design/jaas.ai
# Move static to assets

## Done
To resolve the conflict in URLs between the jaas-dashboard and jaas.ai website. I have moved the static directory to assets and left the jaas-dashboard as static.

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Open the network tab and filter for local assets
- Check they all load from `/assets` instead of `static`
- Check everything still works like copy to clipboard, etc.
